README.md

```diff
@@ -155,3 +155,51 @@ Some tests may require an EasyPost user with a particular set of enabled feature
 - `USPS_CARRIER_ACCOUNT_ID` (eg: one-call buying a shipment for non-EasyPost employees)
 - `PARTNER_USER_PROD_API_KEY` (eg: creating a referral user)
 - `REFERRAL_USER_PROD_API_KEY` (eg: adding a credit card to a referral user)
+
+#### Mocking
+
+Some ouf our unit tests requiring HTTP calls that cannot be easily tested with live/recorded calls (e.g. HTTP calls that
+trigger payments or interact with external APIs).
+
+We have implemented a custom, lightweight HTTP mocking functionality in this library that allows us to mock HTTP calls
+and responses.
+
+The mock client is the same as a normal client, with a set of mock request-response pairs stored as a property.
+
+At the time to make a real HTTP request, the mock client will instead check which mock request entry matches the queued
+request (matching by HTTP method and a regex pattern for the URL), and will return the corresponding mock response (HTTP
+status code and body).
+
+**NOTE**: If a client is configured with any mock request entries, it will ONLY make mock requests. If it attempts to
+make a request that does not match any of the configured mock requests, the request will fail and trigger an exception.
+
+To use the mock client:
+
+```go
+import "github.com/EasyPost/easypost-go/v2"
+
+// create  a list of mock request-response pairs
+mockRequests := []easypost.MockRequest{
+	{
```

Comment:
Huh, something in this block still looks like the indentation is off? Can you confirm? Maybe it's only this opening line?

Reply:
Had to tab/space manually, because my IDE hates go in READMEs, I guess.
<img width="509" alt="image" src="https://user-images.githubusercontent.com/17054780/205720182-df1486da-3154-43f9-b561-4eefccf7bd63.png">